infrastructure/variables.tf

```diff
@@ -74,6 +74,11 @@ variable "file_cleanup_enabled" {
   description = "Determines whether old files should be deleted from the FTP server. Should only be enabled on AAT."
 }
 
+variable "file_cleanup_cron" {
+  default     = "0 0 5 * * *"
```

Comment:
You have defined the default here which will cause to run remove files on prod ?

Reply:
Default value is already present in main application config. Enable/Disable flag is above and it will be disabled for prod as it now